packages/ove-service-imagetiles/src/OVE.Service.ImageTiles/Startup.cs

```diff
@@ -155,6 +155,7 @@ public class Startup {
             app.UseSwagger()
                 .UseSwaggerUI(c => {
                     c.SwaggerEndpoint("/swagger/" + _version + "/swagger.json", "TileService " + _version);
+					c.RoutePrefix = "api-docs";

```

Comment:
This line is indented with 5 tabs (rather than 5 sets of 4 spaces). It can therefor look misaligned depending on how an editor displays spaces (e.g. in the GitHub preview above it is indented by too much).

```suggestion
                    c.RoutePrefix = "api-docs";
```